packages/patternfly-4/react-core/src/components/Page/Page.tsx

```diff
@@ -147,7 +155,13 @@ export class Page extends React.Component<PageProps, PageState> {
           {skipToContent}
           {header}
           {sidebar}
-          <main role="main" id={mainContainerId} className={css(styles.pageMain)} tabIndex={-1} aria-label={mainAriaLabel}>
+          <main
+            role={isRoleSet ? role : undefined}
```

Comment:
I'm confused as to why isRoleSet is necessary as a prop? Considering role is undefined by default, there wouldn't need to test it here. I don't see an example how this is used, unless I'm missing something?

Reply:
Yes @dlabrecq that makes sense, you're right. We'd probably never need the prop so I removed it -- thank you for clarifying